src/main/java/org/testeditor/fixture/host/HostDriverFixture.java

```diff
@@ -136,6 +142,68 @@ public void sendControlCommand(ControlCommand value) {
         connection.doCommand("ascii"); // just to see if typed in successfully.
     }
 
+    /**
+     * Reads a value on a given position range start(x,y) and end(x,y)
+     * 
+     * @param locator
+     *            The Locator exists of starting(row,column) and end(row,column)
+     *            in the form "2;12;5;16" which means the start position is at
+     *            row = 2, column = 12 up to stop position row = 5, column 16.
+     * @return the String value which is read at the given position.
+     */
+
+    /**
+     * Reads a value on a given position range start(x,y) and end(x,y)
+     * 
+     * @param elementLocator
+     *            The elementlocator exists of position definitions with a
+     *            starting(row,column) and end(row,column) position in the form
+     *            "2;12;5;16" which means the start position is at row = 2,
```

Comment:
That should not take longer than an hour and would make it more readable for the user, wouldn't it? There is a good website for developing regular expressions: http://regexr.com/

So for `2;12-5;16` it would be `\d+;\d+-\d+;\d+`